plugins/Annotations/Model.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ *
+ */
+
+namespace Piwik\Plugins\Annotations;
+
+use Piwik\Common;
+use Piwik\Db;
+
+class Model
+{
+    private static $rawPrefix = 'annotations';
```

Comment:
```suggestion
    private const TABLE_NAME = 'annotations';
```

The wording (and even being a constant) is personal taste, but it should say we are looking at athe raw *unprefixed* table name.

Reply:
Replaced with a CONST.